indexing-service/src/main/java/org/apache/druid/indexing/overlord/sampler/SamplerConfig.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.overlord.sampler;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SamplerConfig
+{
+  private static final int DEFAULT_NUM_ROWS = 200;
+  private static final boolean DEFAULT_SKIP_CACHE = false;
+  private static final int DEFAULT_TIMEOUT_MS = 10000;
+
+  private final int numRows;
```

Comment:
Would you please add a comment about that the actual number of sampled rows would be `min(number of cached rows, numRows)` if `cacheKey` is valid and `skipCache` is false?